src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -33,3 +37,5 @@ The divider renders as an `<hr>` by default. It is possible to make the divider
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-c-divider` | `<hr>`, `<li>`, `<div>` | Defines the divider component. |
+| `.pf-m-vertical` | `.pf-c-divider` | Modifies the divider component from horizontal to vertical. |
+| `.pf-m-stretch` | `.pf-m-vertical` | Modifies the vertical divider to stretch to full height. This modifier requires that the parent has an explicit or implicit height value. |
```

Comment:
I think "This modifier requires that the parent has an explicit or implicit height value" belongs in the previous line (`.pf-c-divider.pf-m-vertical`) - the base variation is the standard inline/inline-block use that requires a height value. `.pf-m-stretch` removes `height`, so we should document it's use is required to be used as the child of a flex parent.